src/history/hs.c

```diff
@@ -355,7 +355,8 @@ __hs_insert_record_with_btree(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_BT
      */
     cursor->set_key(
       cursor, btree->id, key, upd->start_ts, __wt_atomic_add64(&btree->hs_counter, 1));
-    cursor->set_value(cursor, stop_ts_pair.timestamp, upd->durable_ts, type, hs_value);
+    cursor->set_value(
+      cursor, stop_ts_pair.timestamp, upd->durable_ts, type, hs_value, (uint8_t)0, (uint64_t)0);
```

Comment:
When I was testing, the casting seemed necessary to get this to work which I thought was pretty error-prone... has anyone else run into this?